packages/headless/package.json

```diff
@@ -160,7 +160,7 @@
   },
   "dependencies": {
     "@coveo/bueno": "1.0.11",
-    "@coveo/relay": "1.0.0",
```

Comment:
not sure why we're pinning to specific versions here rather than using ^, but following the convention

Reply:
That's not an "official" version though, not tagged latest on npm 🤔